cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgisSink.java

```diff
@@ -549,16 +549,18 @@ public void aggregate(NGSIEvent cygnusEvent) {
                 } else {
                     // create part of the column with the current attribute (a.k.a. a column)
                     if (attrNativeTypes) {
-                        if (attrType.equals("Number")) {
-                            column += "," + attrValue + ",'"  + attrMetadata + "'";
-                        } else {
-                            if (attrValue == null || attrValue.equals("")) {
-                                attrValue = "NULL";
+                        if (contextAttribute.getValue().isJsonNull()) {
+                            column += ",NULL,'" + attrMetadata + "'";
```

Comment:
If no attribute value, for sure there is no attrMetadata. Please, correct.

On the other hand, we should deal with the problem of missing attributes in batch insertions. Missing attributes should be included as NULL

Reply:
Fixed in [1f72cde](https://github.com/telefonicaid/fiware-cygnus/pull/1784/commits/1f72cde9337c8b9cd68d42a86307cbade02acff2) and WIP respectively.

Reply:
> On the other hand, we should deal with the problem of missing attributes in batch insertions. Missing attributes should be included as NULL

This only applies to column mode?